### PR TITLE
Fix test release 015

### PR DESCRIPTION
**Pull request by Jillderon**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/1306

### CODE REVIEW

**Review by TimKDJ — CHANGES_REQUESTED** [review]
This does not fix the linked issue; the plots are still re-made -- you need to look at `.equivalencePlotInd()` (and possibly other plotting functions)?

File: R/equivalencebayesianpairedsamplesttest.R [hunk 1]
```diff
@@ -44,7 +44,7 @@ EquivalenceBayesianPairedSamplesTTest <- function(jaspResults, dataset, options)
   # Output tables and plots
   .equivalenceBayesianPairedTTestTableMain(jaspResults, dataset, options, equivalenceBayesianPairedTTestResults, ready)
 
-  if(options$descriptives)
+  if(options$descriptives && is.null(jaspResults[["equivalenceBayesianDescriptivesTable"]]))
```
[the remark this answers is not in the document]
Comment (TimKDJ) [reply R/equivalencebayesianpairedsamplesttest.R:47]: Actually [that function already does a null check](https://github.com/jasp-stats/jaspEquivalenceTTests/blob/986d23fbc80d6dea7f1403097fce006582fb0ae1/R/equivalencebayesianpairedsamplesttest.R#L220), so this addition is not necessary

**Comment by Kucharssim** [comment]
@Jillderon do you want to work this in 0.16.3?

**Comment by FBartos** [comment]
this is obsolete now

**Pull request closed without merging**